.gitignore

```diff
@@ -193,6 +193,7 @@ src/tools/printjob.bin
 src/tools/printjob_svr.bin
 src/tools/tracejob
 src/tools/wrap_tcl.sh
+src/tools/pbs_sleep
```

Comment:
Check if this is already there.  I added pbs_sleep to the .gitignore file a couple of weeks ago.